.github/workflows/dart_frog_prod_server.yaml

```diff
@@ -14,8 +14,9 @@ on:
 
 jobs:
   build:
-    uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
+    uses: ./.github/workflows/.dart_package.yaml
```

Comment:
Any reason that this is still needed?

Reply:
No I need to revert and figure out why coverage is behaving differently in Linux vs MacOS